drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-businesscentral-client/src/main/java/org/drools/workbench/screens/scenariosimulation/businesscentral/client/editor/strategies/BusinessCentralDMODataManagementStrategy.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright 2018 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.drools.workbench.screens.scenariosimulation.businesscentral.client.editor.strategies;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.SortedMap;
+
+import org.drools.workbench.screens.scenariosimulation.client.commands.ScenarioSimulationContext;
+import org.drools.workbench.screens.scenariosimulation.client.editor.strategies.AbstractDMODataManagementStrategy;
+import org.drools.workbench.screens.scenariosimulation.client.models.ScenarioGridModel;
+import org.drools.workbench.screens.scenariosimulation.client.rightpanel.TestToolsView;
+import org.drools.workbench.screens.scenariosimulation.model.ScenarioSimulationModelContent;
+import org.drools.workbench.screens.scenariosimulation.model.typedescriptor.FactModelTree;
+import org.kie.soup.project.datamodel.oracle.ModelField;
+import org.kie.workbench.common.widgets.client.datamodel.AsyncPackageDataModelOracle;
+import org.kie.workbench.common.widgets.client.datamodel.AsyncPackageDataModelOracleFactory;
+import org.uberfire.backend.vfs.ObservablePath;
+import org.uberfire.client.callbacks.Callback;
+
+public class BusinessCentralDMODataManagementStrategy extends AbstractDMODataManagementStrategy {
+
+    private AsyncPackageDataModelOracleFactory oracleFactory;
+    protected AsyncPackageDataModelOracle oracle;
+
+    //Package for which this Scenario Simulation relates
+    protected String packageName = "";
```

Comment:
It seems this is unused, is it correct? Can we remove it?

Reply:
@yesamer 
Yes - could be removed

Reply:
Done.